signing-providers/src/web-wallet.js

```diff
@@ -32,6 +34,14 @@ export class WebWallet {
         alert(getUrlParams().signature || "Try to login (with token) first.");
     }
 
+    validateTokenSignature() {
+        const address = getUrlParams().address;
+        const authToken = sessionStorage.getItem("web-wallet-example:authToken");
```

Comment:
Can we use await here, before the sessionStorage.getItem? If you run this on React Native and polyfill the sessionStorage with AsyncStorage, it works on a promise base, and this will fail. On web, this will execute normally, the await will just be ignored

Reply:
Fixed, and added a disclaimer.